cluster/gce/gci/master.yaml

```diff
@@ -58,12 +59,13 @@ write_files:
     content: |
       [Unit]
       Description=Configure kube internal route
+      Requires=kube-master-installation.service
       After=kube-master-installation.service
 
       [Service]
       Type=oneshot
       RemainAfterExit=yes
-      ExecStartPre=/usr/bin/curl --fail --retry 5 --retry-delay 3 --silent --show-error -H "X-Google-Metadata-Request: True" -o /home/kubernetes/bin/kube-master-internal-route.sh http://metadata.google.internal/computeMetadata/v1/instance/attributes/kube-master-internal-route
+      ExecStartPre=/usr/bin/curl --fail --retry 5 --retry-delay 3 --retry-connrefused --silent --show-error -H "X-Google-Metadata-Request: True" -o /home/kubernetes/bin/kube-master-internal-route.sh http://metadata.google.internal/computeMetadata/v1/instance/attributes/kube-master-internal-route
```

Comment:
Should this be --retry 600 as well?

Reply:
I had increased kube-master-installation only, because it is the hub unit which all these others depend on, and we had real occurrences of it failing. I could bump these as well, but I suspect that when GCE metadata becomes available for kube-master-installation it will for these too.

Maybe a middle number, like 100?

Reply:
Nah, let's leave it - your reasoning is sound.